released/packages/coq-verified-extraction/coq-verified-extraction.0.9.1+8.19/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Matthieu Sozeau <matthieu.sozeau@inria.fr"
+homepage: "https://github.com/yforster/coq-verified-extraction"
+dev-repo: "git+https://github.com/yforster/coq-verified-extraction"
+bug-reports: "https://github.com/yforster/coq-verified-extraction/issues"
+authors:  [
+          "Yannick Forster"
+          "Matthieu Sozeau"
+          "Nicolas Tabareau"
+  ]
+license: "MIT"
+build: [
+  [make]
```

Comment:
```suggestion
  [make "-j%{jobs}%"]
```
@mattam82 is there a reason not to allow parallelism here?

Reply:
A priori, no there is no reason